### PR TITLE
os: Keep Docker script in the menu loop even after downloading

### PR DESCRIPTION
**Pull request by amogh-samsung**
- After chosing to Erase a partition, the user may wish to download/erase another one.
- After downloading a particular partition, the user may wish to download yet another one.
- This gives the user the option to continue working with docker after any download command completes.
- Also, changes the STATUS option from "DOWNLOAD" to "DOWNLOAD READY" to differentiate between the status and a script function.

Signed-off-by: Amogh Hassija <a.hassija@samsung.com>

### CODE REVIEW

File: os/dbuild.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ BINDIR="${BUILDDIR}/output/bin"
 CONFIGDIR="${BUILDDIR}/configs"
 DOCKER_VERSION="1.5.5"
 
-STATUS_LIST="NOT_CONFIGURED BOARD_CONFIGURED CONFIGURED BUILT PREPARE_DL DOWNLOAD"
+STATUS_LIST="NOT_CONFIGURED BOARD_CONFIGURED CONFIGURED BUILT PREPARE_DL DOWNLOAD_READY"
```
Comment (kishore-sn) [inline os/dbuild.sh:29]: Can you pls check all the other places in this file where "`DOWNLOAD`" is used and change it accordingly?

Comment (amogh-samsung) [reply os/dbuild.sh:29]: Checked and updated

File: os/dbuild.sh [hunk 2]
```diff
@@ -26,7 +26,7 @@ BINDIR="${BUILDDIR}/output/bin"
 CONFIGDIR="${BUILDDIR}/configs"
```
Comment (abhishek-samsung) [inline os/dbuild.sh:26]: Please change "chosing" to "choosing" in the commit message.

Comment (amogh-samsung) [reply os/dbuild.sh:26]: Done

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**